server/issue.go

```diff
@@ -370,6 +371,25 @@ func httpAPIAttachCommentToIssue(ji Instance, w http.ResponseWriter, r *http.Req
 
 	commentAdded, _, err := jiraClient.Issue.AddComment(attach.IssueKey, &jiraComment)
 	if err != nil {
+		if strings.Contains(err.Error(), "you do not have the permission to comment on this issue") {
```

Comment:
This LGTM, but I wonder if there's a more "future-proof" way of doing this check by decoding the response' body, as JSON?

Reply:
checking...

Reply:
@levb It looks like go-jira only returns the error, it doesn't pass along the response... And the error looks to just be a string...  In the Jira API it's listed as `A schema has not been defined for this response code.`